tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -6,12 +6,15 @@
     PandasDataFrameDatasourceTestConfig,
     PandasFilesystemCsvDatasourceTestConfig,
 )
+from tests.integration.test_utils.data_source_config.snowflake import NUMBER as SNOWFLAKE_NUMBER
```

Comment:
It'd be nice to not have to alias. Maybe better to do a `import snowflake.sqlalchemy.custom_types as SNOWFLAKE_TYPES` - or could we just export that object from our config file?

Reply:
we're thinking the same thoughts 🙌